config/settings.yml

```diff
@@ -1,23 +1,23 @@
 multitenancy:
   enabled: false
-  default_host: "<%= ENV.fetch('SETTINGS__MULTITENANCY__DEFAULT_HOST', '%{tenant}.dev') %>"
+  default_host: "%{tenant}.dev"
```

Comment:
Would it be helpful to write something either as a comment in this file, or in the readme explaining how use env variables to override the settings here?